scripts/k8s/deploy_ingress.sh

```diff
@@ -55,5 +55,5 @@ fi
 # Set up the ingress controller
 if ! helm status "${app_name}" >/dev/null 2>&1; then
 	helm repo update
-	helm install --wait "${app_name}" "${helm_arguments[@]}" ingress-nginx/ingress-nginx --create-namespace --namespace deepops-ingress
+	helm upgrade --install --wait "${app_name}" "${helm_arguments[@]}" ingress-nginx/ingress-nginx --create-namespace --namespace deepops-ingress
```

Comment:
This is required for a section in the test that was previously silently failing due to an existing ingress install that was not properly deleted. This is also the mechanism we have been moving all our other helm installs to use.